### ISSUE TITLE
Wrong copyright text

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#861]
**Describe the bug**
The design has differing copywriting regarding the Footer copyright text. Design: "© 2018 BBC", Current Test: `Copyright © 2018 BBC`. This means that currently 'copyright' is read out twice when using a screen reader.

**To Reproduce**
View the copyright text in the Footer, and with a screen reader.

**Expected behaviour**
The copyright should be `© 2018 BBC`
